MovieApp/MovieApp/App/Features/Movies/UseCases/MoviesUseCase.swift

```diff
@@ -6,8 +6,22 @@ class MoviesUseCase: MoviesUseCaseProtocol {
         self.repository = repository
     }
     
-    func fetchMovies(completion: @escaping (NetworkResult<MoviesNetworkModel, NetworkError>) -> Void) {
-        repository.fetchMovies(completion: completion)
+    func fetchMovies(completion: @escaping (NetworkResult<[MovieModel], NetworkError>) -> Void) {
+        repository.fetchMovies { (result: NetworkResult<[MovieRepositoryModel], NetworkError>) in
+            switch result {
+            case .failure(let error):
+                print(error.localizedDescription)
+            case .success(let value):
+                let useCaseModel: [MovieModel] = value.map { model -> MovieModel in
+                    let imageUrl = NetworkConstants.imagePath + model.imageUrl
```

Comment:
why do you also have this in repo layer?

Reply:
Accidentally forgot to remove it from repo layer.